src/app/Agent/AgentLiveProbes.tsx

```diff
@@ -0,0 +1,266 @@
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import * as React from 'react';
+import { ServiceContext } from '@app/Shared/Services/Services';
+import { NotificationCategory } from '@app/Shared/Services/NotificationChannel.service';
+import { NO_TARGET } from '@app/Shared/Services/Target.service';
+import { useSubscriptions } from '@app/utils/useSubscriptions';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeaderMain,
+  CardHeader,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+  TextInput,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableVariant,
+  ISortBy,
+  SortByDirection,
+  sortable,
+} from '@patternfly/react-table';
+import { useHistory } from 'react-router-dom';
+import { concatMap, filter, first } from 'rxjs/operators';
+import { LoadingView } from '@app/LoadingView/LoadingView';
+import { ErrorView, isAuthFail } from '@app/ErrorView/ErrorView';
+import { EventProbe } from '@app/Shared/Services/Api.service';
+
+export const AgentLiveProbes = () => {
+  const context = React.useContext(ServiceContext);
+  const history = useHistory();
+
+  const [templates, setTemplates] = React.useState([] as EventProbe[]);
+  const [filteredTemplates, setFilteredTemplates] = React.useState([] as EventProbe[]);
+  const [filterText, setFilterText] = React.useState('');
+  const [sortBy, setSortBy] = React.useState({} as ISortBy);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [errorMessage, setErrorMessage] = React.useState('');
+  const addSubscription = useSubscriptions();
+
+  const tableColumns = [
+    { title: 'ID', transforms: [sortable] },
+    { title: 'Label', transforms: [sortable] },
+    { title: 'Class', transforms: [sortable] },
+    { title: 'Description', transforms: [sortable] },
+    { title: 'Method', transforms: [sortable] },
+  ];
+
+  React.useEffect(() => {
+    let filtered;
+    if (!filterText) {
+      filtered = templates;
+    } else {
+      const ft = filterText.trim().toLowerCase();
+      filtered = templates.filter(
+        (t: EventProbe) =>
+          t.name.toLowerCase().includes(ft) ||
+          t.description.toLowerCase().includes(ft) ||
+          t.clazz.toLowerCase().includes(ft) ||
+          t.methodDescriptor.toLowerCase().includes(ft) ||
+          t.methodName.toLowerCase().includes(ft)
+      );
+    }
+    const { index, direction } = sortBy;
+    if (typeof index === 'number') {
+      const keys = ['ID', 'Label', 'Description', 'Class', 'Method'];
```

Comment:
Keys should match case of the object fields.

https://github.com/Josh-Matsuoka/cryostat-web/blob/a74db433295f7edb2c310486b484e03e41954c0a/src/app/Shared/Services/Api.service.tsx#L1058

Looks like we are using `Label` column for name. Should be change to just `Name`?